core/include/scattered_array.h

```diff
@@ -0,0 +1,131 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+#ifndef __SCATTERED_ARRAY_H
+#define __SCATTERED_ARRAY_H
+
+#include <compiler.h>
+#include <keep.h>
+
+/*
+ * A scattered array is assembled from items declared in different source
+ * files using depending on something like "SORT(.scattered_array*)" in the
```

Comment:
`using depending on something like ...`  ???
maybe choose betwen 'using' and 'depending on'.

Reply:
Thanks, I'll fix.